webapp/src/widgets/label.tsx

```diff
@@ -13,9 +15,13 @@ type Props = {
 
 // Switch is an on-off style switch / checkbox
 function Label(props: Props): JSX.Element {
+    let color = 'empty'
+    if (Constants.menuColors.hasOwnProperty(props.color || '')) {
```

Comment:
Won't `Constants.menuColors.hasOwnProperty('')` always return false? You already init with `let color = 'empty'` so I don't see this necessary to check.

Reply:
This is because `hasOwnProperty` requires an string and props.color can be undefined.